iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -891,6 +906,18 @@ struct MakeDispatchWorkgroupsOp : public RewritePattern {
       return failure();
     }
 
+    // If this is a standalone fill op, we don't need to create a dispatch
+    // region for it; just use flow.tensor.splat so we can leverage DMA
+    // functionalities.
+    Location loc = op->getLoc();
+    if (auto fillOp = dyn_cast<linalg::FillOp>(op)) {
```

Comment:
This doesnt seem to be the right place to do this. Its not actually creating a dispatch region :) . 

Maybe move this to `ConvertToFlowTensorOps` pass. Make that pass operate in two modes, before dispatch region creation and after (using a flag). The same pass can be run before and after dispatch region creation. This `fill` conversion will be added to the "after" path.

Reply:
Yup. I was in the middle of addressing this comment. (That's why I haven't re-requested review. ;-P)

It's fine for me to move to `ConvertToFlowTensorOps.cpp` and actually it results in slightly better code structure I think. But one concern I have is that it's assuming all `linalg.fill` ops with linalg users will be fused down the pipeline, while this is more generic as catch all sink. But the assumption is probably true as I cannot immediately come up with a case where it's not. So moved to `ConvertToFlowTensorOps.cpp`. :)

Reply:
Ah, I think Github sent me notification and I thought this was ready for review. But still have some comments below.